providers/parameter.rb

```diff
@@ -51,6 +51,7 @@ def parameter_exists?(vhost, name)
 
     execute "set_parameter #{parameter}" do
       command cmd
+      environment 'HOME' => ENV.fetch('HOME', '/root')
```

Comment:
If so, should I change it to that in the places where `/root` was already being set as well?